app/models/agents/chattermill_response_agent.rb

```diff
@@ -51,6 +51,7 @@ class ChattermillResponseAgent < Agent
           * `segments` - Specify the Liquid interpolated JSON to build the Response segments.
           * `extra_fields` - Specify the Liquid interpolated JSON to build additional fields for the Response, e.g: `{ approved: true }`.
           * `mappings` - Specify the mapping definition object where any field can be mapped with a single value.
+          * `bucketing_object` - Specify the bucketing definition object where any field can be broken into a specific bucket.
```

Comment:
@monsha Forgot to ask, why is this called `bucketing_object` but the options item is called `bucketing`